### PR TITLE
Cap each log field (message, stacktrace) to max 32K characters

### LINKED ISSUE #30781: 🐛 Large log message in java connectors can lead to loss of records
### Topic
Unsafe logging can lead to loss of AirbyteMessages in java connectors

### Revelant information

This is an issue observed in source-postgres, but the issue is low level and shared among all java connectors.

A log message that is:
a. longer than 8KB
b. sent from a secondary thread.

May be written partially to stdout and in case another AirbyteMessage such as a `Record` is written at the same time, may cause both message interleaving.
In the case of source this may lead to a loss of records as stdout emits
`{"type":"LOG","log":{"level":"INFO","message":"partial log line … {"type":"RECORD","record":…}`
Because the two interleaved messages are written in a single line, the receiver of the message - platform orchestrator - cannot decode a valid json and the messages get omitted.
On source this will lead to a loss of record
On destination this may lead to a loss of a checkpoint ack for example.

See [repro code](https://github.com/airbytehq/airbyte/commit/4c758df35c188bea2b0e39f2ecc2fd7730500a5e)
Running sync with source postgres will cause a loss of records.

A real [connection](https://cloud.airbyte.com/workspaces/b79f07e0-0575-4ed5-93bb-6ef9c6396646/connections/ba7b1ada-7168-4ef6-9df6-bffb7e20aa46/job-history#4770161::0) in which we see the issue

### PR DESCRIPTION
**Pull request by rodireich**
## What
A thread safety issue causes log lines to be interleaved with other `AirbyteMessage`s when:
1. logging from another thread
2. Log line is over 8K characters

## How
Cap max log line length to undo 8k by limiting both message size and logged stack trace size to under 16,000 characters

### CODE REVIEW

File: airbyte-integrations/bases/base-java/src/main/resources/log4j2.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
   <Appenders>
     <Console name="ConsoleJSONAppender" target="SYSTEM_OUT">
-      <JsonTemplateLayout eventTemplateUri="classpath:AirbyteLogMessageTemplate.json"/>
+      <JsonTemplateLayout eventTemplateUri="classpath:AirbyteLogMessageTemplate.json" maxStringLength="3750"/>/>
```
Comment (tryangul) [inline airbyte-integrations/bases/base-java/src/main/resources/log4j2.xml:6]: Can we leave a comment here or in the code explaining the significance of this number?

Comment (rodireich) [reply airbyte-integrations/bases/base-java/src/main/resources/log4j2.xml:9]: Added

**Review by prateekmukhedkar — APPROVED** [review]

**Review by evantahler — APPROVED** [review]

File: airbyte-integrations/bases/base-java/javabase.sh [hunk 2]
```diff
@@ -15,6 +15,9 @@ if [[ $IS_CAPTURE_HEAP_DUMP_ON_ERROR = true ]]; then
       echo "APPLICATION=$APPLICATION"
   fi
 fi
+#30781 - Allocate 32KB for log4j appender buffer to ensure that each line is logged in a single println
+JAVA_OPTS=$JAVA_OPTS" -Dlog4j.encoder.byteBufferSize=32768"
```
Comment (rodireich) [inline airbyte-integrations/bases/base-java/javabase.sh:19]: This control the size of buffer defined in log4j's ConsoleAppender
(See [here](https://logging.apache.org/log4j/log4j-2.17.1/log4j-core/apidocs/src-html/org/apache/logging/log4j/core/appender/AbstractOutputStreamAppender.html) line 47)

This is the base shell command for all java connector.
We could put this JVM options separately on each connector's build.gradle but it's likely to be forgotten.
@davinchia @evantahler @prateekmukhedkar

**Comment by airbyte-oss-build-runner** [comment]
##  source-postgres-strict-encrypt test report (commit [`2c6028b9db`](https://github.com/airbytehq/airbyte/pull/30806/commits/2c6028b9db6e60e03d631100ff415177f2d3fbf6)) - ❌

⏲️  Total pipeline duration: 11mn02s 

| Step                                                                        | Result   |
|:----------------------------------------------------------------------------|:---------|
| Build connector tar                                                         | ✅        |
| Build source-postgres-strict-encrypt docker image for platform linux/x86_64 | ✅        |
| Java Connector Unit Tests                                                   | ✅        |
| Java Connector Integration Tests                                            | ❌        |
| Acceptance tests                                                            | ✅        |
| Validate metadata for source-postgres-strict-encrypt                        | ✅        |
| Connector version semver check                                              | ✅        |
| Connector version increment check                                           | ✅        |
| QA checks                                                                   | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/30781-truncate-log/1696008236/2c6028b9db6e60e03d631100ff415177f2d3fbf6/source-postgres-strict-encrypt/3.1.10/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:2c6028b9db6e60e03d631100ff415177f2d3fbf6)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-postgres-strict-encrypt test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-postgres test report (commit [`2c6028b9db`](https://github.com/airbytehq/airbyte/pull/30806/commits/2c6028b9db6e60e03d631100ff415177f2d3fbf6)) - ❌

⏲️  Total pipeline duration: 26mn34s 

| Step                                                         | Result   |
|:-------------------------------------------------------------|:---------|
| Build connector tar                                          | ✅        |
| Build source-postgres docker image for platform linux/x86_64 | ✅        |
| Java Connector Unit Tests                                    | ✅        |
| Java Connector Integration Tests                             | ❌        |
| Acceptance tests                                             | ✅        |
| Validate metadata for source-postgres                        | ✅        |
| Connector version semver check                               | ✅        |
| Connector version increment check                            | ✅        |
| QA checks                                                    | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/30781-truncate-log/1696008236/2c6028b9db6e60e03d631100ff415177f2d3fbf6/source-postgres/3.1.10/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:2c6028b9db6e60e03d631100ff415177f2d3fbf6)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-postgres test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-alloydb-strict-encrypt test report (commit [`2c6028b9db`](https://github.com/airbytehq/airbyte/pull/30806/commits/2c6028b9db6e60e03d631100ff415177f2d3fbf6)) - ❌

⏲️  Total pipeline duration: 03mn40s 

| Step                                                                       | Result   |
|:---------------------------------------------------------------------------|:---------|
| Build connector tar                                                        | ✅        |
| Build source-alloydb-strict-encrypt docker image for platform linux/x86_64 | ✅        |
| Java Connector Unit Tests                                                  | ✅        |
| Java Connector Integration Tests                                           | ✅        |
| Acceptance tests                                                           | ❌        |
| Validate metadata for source-alloydb-strict-encrypt                        | ✅        |
| Connector version semver check                                             | ✅        |
| Connector version increment check                                          | ✅        |
| QA checks                                                                  | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/30781-truncate-log/1696008236/2c6028b9db6e60e03d631100ff415177f2d3fbf6/source-alloydb-strict-encrypt/3.1.10/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:2c6028b9db6e60e03d631100ff415177f2d3fbf6)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-alloydb-strict-encrypt test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-mysql test report (commit [`2c6028b9db`](https://github.com/airbytehq/airbyte/pull/30806/commits/2c6028b9db6e60e03d631100ff415177f2d3fbf6)) - ❌

⏲️  Total pipeline duration: 24mn49s 

| Step                                                      | Result   |
|:----------------------------------------------------------|:---------|
| Build connector tar                                       | ✅        |
| Build source-mysql docker image for platform linux/x86_64 | ✅        |
| Java Connector Unit Tests                                 | ❌        |
| Java Connector Integration Tests                          | ❌        |
| Acceptance tests                                          | ✅        |
| Validate metadata for source-mysql                        | ✅        |
| Connector version semver check                            | ✅        |
| Connector version increment check                         | ✅        |
| QA checks                                                 | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/30781-truncate-log/1696008236/2c6028b9db6e60e03d631100ff415177f2d3fbf6/source-mysql/3.1.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:2c6028b9db6e60e03d631100ff415177f2d3fbf6)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mysql test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-mysql-strict-encrypt test report (commit [`2c6028b9db`](https://github.com/airbytehq/airbyte/pull/30806/commits/2c6028b9db6e60e03d631100ff415177f2d3fbf6)) - ❌

⏲️  Total pipeline duration: 08mn23s 

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Build connector tar                                                      | ✅        |
| Build source-mysql-strict-encrypt docker image for platform linux/x86_64 | ✅        |
| Java Connector Unit Tests                                                | ❌        |
| Java Connector Integration Tests                                         | ❌        |
| Acceptance tests                                                         | ✅        |
| Validate metadata for source-mysql-strict-encrypt                        | ✅        |
| Connector version semver check                                           | ✅        |
| Connector version increment check                                        | ✅        |
| QA checks                                                                | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/30781-truncate-log/1696008236/2c6028b9db6e60e03d631100ff415177f2d3fbf6/source-mysql-strict-encrypt/3.1.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:2c6028b9db6e60e03d631100ff415177f2d3fbf6)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mysql-strict-encrypt test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-alloydb test report (commit [`2c6028b9db`](https://github.com/airbytehq/airbyte/pull/30806/commits/2c6028b9db6e60e03d631100ff415177f2d3fbf6)) - ❌

⏲️  Total pipeline duration: 04mn01s 

| Step                                                        | Result   |
|:------------------------------------------------------------|:---------|
| Build connector tar                                         | ✅        |
| Build source-alloydb docker image for platform linux/x86_64 | ✅        |
| Java Connector Unit Tests                                   | ✅        |
| Java Connector Integration Tests                            | ✅        |
| Acceptance tests                                            | ❌        |
| Validate metadata for source-alloydb                        | ✅        |
| Connector version semver check                              | ✅        |
| Connector version increment check                           | ✅        |
| QA checks                                                   | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/30781-truncate-log/1696008236/2c6028b9db6e60e03d631100ff415177f2d3fbf6/source-alloydb/3.1.10/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:2c6028b9db6e60e03d631100ff415177f2d3fbf6)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-alloydb test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-postgres-strict-encrypt test report (commit [`40d3f99ca5`](https://github.com/airbytehq/airbyte/pull/30806/commits/40d3f99ca5855b437d40da979868056af0666280)) - ❌

⏲️  Total pipeline duration: 10mn33s 

| Step                                                                           | Result   |
|:-------------------------------------------------------------------------------|:---------|
| Build connector tar                                                            | ✅        |
| Build source-postgres-strict-encrypt docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                                      | ✅        |
| Java Connector Integration Tests                                               | ❌        |
| Acceptance tests                                                               | ✅        |
| Validate metadata for source-postgres-strict-encrypt                           | ✅        |
| Connector version semver check                                                 | ✅        |
| QA checks                                                                      | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/30781-truncate-log/1697042053/40d3f99ca5855b437d40da979868056af0666280/source-postgres-strict-encrypt/3.1.10/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:40d3f99ca5855b437d40da979868056af0666280)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-postgres-strict-encrypt test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-pokeapi test report (commit [`884985241e`](https://github.com/airbytehq/airbyte/pull/30806/commits/884985241e54f91ac3abc207521af13e43eb3213)) - ✅

⏲️  Total pipeline duration: 01mn13s 

| Step                                                           | Result   |
|:---------------------------------------------------------------|:---------|
| Build source-pokeapi docker image for platform(s) linux/x86_64 | ✅        |
| Acceptance tests                                               | ✅        |
| Code format checks                                             | ✅        |
| Validate metadata for source-pokeapi                           | ✅        |
| Connector version semver check                                 | ✅        |
| QA checks                                                      | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/30781-truncate-log/1697043254/884985241e54f91ac3abc207521af13e43eb3213/source-pokeapi/0.2.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:884985241e54f91ac3abc207521af13e43eb3213)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-pokeapi test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-mysql test report (commit [`884985241e`](https://github.com/airbytehq/airbyte/pull/30806/commits/884985241e54f91ac3abc207521af13e43eb3213)) - ✅

⏲️  Total pipeline duration: 28mn57s 

| Step                                                         | Result   |
|:-------------------------------------------------------------|:---------|
| Build connector tar                                          | ✅        |
| Build source-mysql docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                    | ✅        |
| Java Connector Integration Tests                             | ✅        |
| Acceptance tests                                             | ✅        |
| Validate metadata for source-mysql                           | ✅        |
| Connector version semver check                               | ✅        |
| Connector version increment check                            | ✅        |
| QA checks                                                    | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/30781-truncate-log/1697043254/884985241e54f91ac3abc207521af13e43eb3213/source-mysql/3.1.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:884985241e54f91ac3abc207521af13e43eb3213)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mysql test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-mysql-strict-encrypt test report (commit [`884985241e`](https://github.com/airbytehq/airbyte/pull/30806/commits/884985241e54f91ac3abc207521af13e43eb3213)) - ✅

⏲️  Total pipeline duration: 08mn31s 

| Step                                                                        | Result   |
|:----------------------------------------------------------------------------|:---------|
| Build connector tar                                                         | ✅        |
| Build source-mysql-strict-encrypt docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                                   | ✅        |
| Java Connector Integration Tests                                            | ✅        |
| Acceptance tests                                                            | ✅        |
| Validate metadata for source-mysql-strict-encrypt                           | ✅        |
| Connector version semver check                                              | ✅        |
| Connector version increment check                                           | ✅        |
| QA checks                                                                   | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/30781-truncate-log/1697043254/884985241e54f91ac3abc207521af13e43eb3213/source-mysql-strict-encrypt/3.1.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:884985241e54f91ac3abc207521af13e43eb3213)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mysql-strict-encrypt test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-postgres-strict-encrypt test report (commit [`884985241e`](https://github.com/airbytehq/airbyte/pull/30806/commits/884985241e54f91ac3abc207521af13e43eb3213)) - ❌

⏲️  Total pipeline duration: 06mn20s 

| Step                                                                           | Result   |
|:-------------------------------------------------------------------------------|:---------|
| Build connector tar                                                            | ✅        |
| Build source-postgres-strict-encrypt docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                                      | ✅        |
| Java Connector Integration Tests                                               | ❌        |
| Acceptance tests                                                               | ✅        |
| Validate metadata for source-postgres-strict-encrypt                           | ✅        |
| Connector version semver check                                                 | ✅        |
| QA checks                                                                      | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/30781-truncate-log/1697043254/884985241e54f91ac3abc207521af13e43eb3213/source-postgres-strict-encrypt/3.1.10/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:884985241e54f91ac3abc207521af13e43eb3213)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-postgres-strict-encrypt test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-pokeapi test report (commit [`dfaadee40b`](https://github.com/airbytehq/airbyte/pull/30806/commits/dfaadee40bd7491957a4a75a194e767a2c1203c7)) - ✅

⏲️  Total pipeline duration: 02mn00s 

| Step                                                           | Result   |
|:---------------------------------------------------------------|:---------|
| Build source-pokeapi docker image for platform(s) linux/x86_64 | ✅        |
| Acceptance tests                                               | ✅        |
| Code format checks                                             | ✅        |
| Validate metadata for source-pokeapi                           | ✅        |
| Connector version semver check                                 | ✅        |
| QA checks                                                      | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/30781-truncate-log/1697047168/dfaadee40bd7491957a4a75a194e767a2c1203c7/source-pokeapi/0.2.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:dfaadee40bd7491957a4a75a194e767a2c1203c7)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-pokeapi test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-postgres test report (commit [`dfaadee40b`](https://github.com/airbytehq/airbyte/pull/30806/commits/dfaadee40bd7491957a4a75a194e767a2c1203c7)) - ✅

⏲️  Total pipeline duration: 28mn01s 

| Step                                                            | Result   |
|:----------------------------------------------------------------|:---------|
| Build connector tar                                             | ✅        |
| Build source-postgres docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                       | ✅        |
| Java Connector Integration Tests                                | ✅        |
| Acceptance tests                                                | ✅        |
| Validate metadata for source-postgres                           | ✅        |
| Connector version semver check                                  | ✅        |
| Connector version increment check                               | ✅        |
| QA checks                                                       | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/30781-truncate-log/1697047168/dfaadee40bd7491957a4a75a194e767a2c1203c7/source-postgres/3.1.10/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:dfaadee40bd7491957a4a75a194e767a2c1203c7)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-postgres test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-postgres-strict-encrypt test report (commit [`dfaadee40b`](https://github.com/airbytehq/airbyte/pull/30806/commits/dfaadee40bd7491957a4a75a194e767a2c1203c7)) - ✅

⏲️  Total pipeline duration: 07mn46s 

| Step                                                                           | Result   |
|:-------------------------------------------------------------------------------|:---------|
| Build connector tar                                                            | ✅        |
| Build source-postgres-strict-encrypt docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                                      | ✅        |
| Java Connector Integration Tests                                               | ✅        |
| Acceptance tests                                                               | ✅        |
| Validate metadata for source-postgres-strict-encrypt                           | ✅        |
| Connector version semver check                                                 | ✅        |
| QA checks                                                                      | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/30781-truncate-log/1697047168/dfaadee40bd7491957a4a75a194e767a2c1203c7/source-postgres-strict-encrypt/3.1.10/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:dfaadee40bd7491957a4a75a194e767a2c1203c7)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-postgres-strict-encrypt test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-mysql-strict-encrypt test report (commit [`dfaadee40b`](https://github.com/airbytehq/airbyte/pull/30806/commits/dfaadee40bd7491957a4a75a194e767a2c1203c7)) - ✅

⏲️  Total pipeline duration: 03mn45s 

| Step                                                                        | Result   |
|:----------------------------------------------------------------------------|:---------|
| Build connector tar                                                         | ✅        |
| Build source-mysql-strict-encrypt docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                                   | ✅        |
| Java Connector Integration Tests                                            | ✅        |
| Acceptance tests                                                            | ✅        |
| Validate metadata for source-mysql-strict-encrypt                           | ✅        |
| Connector version semver check                                              | ✅        |
| Connector version increment check                                           | ✅        |
| QA checks                                                                   | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/30781-truncate-log/1697047168/dfaadee40bd7491957a4a75a194e767a2c1203c7/source-mysql-strict-encrypt/3.1.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:dfaadee40bd7491957a4a75a194e767a2c1203c7)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mysql-strict-encrypt test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-mysql test report (commit [`dfaadee40b`](https://github.com/airbytehq/airbyte/pull/30806/commits/dfaadee40bd7491957a4a75a194e767a2c1203c7)) - ✅

⏲️  Total pipeline duration: 03mn40s 

| Step                                                         | Result   |
|:-------------------------------------------------------------|:---------|
| Build connector tar                                          | ✅        |
| Build source-mysql docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                    | ✅        |
| Java Connector Integration Tests                             | ✅        |
| Acceptance tests                                             | ✅        |
| Validate metadata for source-mysql                           | ✅        |
| Connector version semver check                               | ✅        |
| Connector version increment check                            | ✅        |
| QA checks                                                    | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/30781-truncate-log/1697047168/dfaadee40bd7491957a4a75a194e767a2c1203c7/source-mysql/3.1.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:dfaadee40bd7491957a4a75a194e767a2c1203c7)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mysql test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-mysql-strict-encrypt test report (commit [`b74634a7a4`](https://github.com/airbytehq/airbyte/pull/30806/commits/b74634a7a460101e37a6850ba2ef4ad7f62109fc)) - ✅

⏲️  Total pipeline duration: 12mn05s 

| Step                                                                        | Result   |
|:----------------------------------------------------------------------------|:---------|
| Build connector tar                                                         | ✅        |
| Build source-mysql-strict-encrypt docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                                   | ✅        |
| Java Connector Integration Tests                                            | ✅        |
| Acceptance tests                                                            | ✅        |
| Validate metadata for source-mysql-strict-encrypt                           | ✅        |
| Connector version semver check                                              | ✅        |
| Connector version increment check                                           | ✅        |
| QA checks                                                                   | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/30781-truncate-log/1697050898/b74634a7a460101e37a6850ba2ef4ad7f62109fc/source-mysql-strict-encrypt/3.1.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:b74634a7a460101e37a6850ba2ef4ad7f62109fc)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mysql-strict-encrypt test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-mysql test report (commit [`50d8f7664d`](https://github.com/airbytehq/airbyte/pull/30806/commits/50d8f7664daaefdf22109ade5de1877cf00762c3)) - ✅

⏲️  Total pipeline duration: 27mn21s 

| Step                                                         | Result   |
|:-------------------------------------------------------------|:---------|
| Build connector tar                                          | ✅        |
| Build source-mysql docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                    | ✅        |
| Java Connector Integration Tests                             | ✅        |
| Acceptance tests                                             | ✅        |
| Validate metadata for source-mysql                           | ✅        |
| Connector version semver check                               | ✅        |
| Connector version increment check                            | ✅        |
| QA checks                                                    | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/30781-truncate-log/1697051995/50d8f7664daaefdf22109ade5de1877cf00762c3/source-mysql/3.1.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:50d8f7664daaefdf22109ade5de1877cf00762c3)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mysql test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-mysql-strict-encrypt test report (commit [`50d8f7664d`](https://github.com/airbytehq/airbyte/pull/30806/commits/50d8f7664daaefdf22109ade5de1877cf00762c3)) - ✅

⏲️  Total pipeline duration: 08mn39s 

| Step                                                                        | Result   |
|:----------------------------------------------------------------------------|:---------|
| Build connector tar                                                         | ✅        |
| Build source-mysql-strict-encrypt docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                                   | ✅        |
| Java Connector Integration Tests                                            | ✅        |
| Acceptance tests                                                            | ✅        |
| Validate metadata for source-mysql-strict-encrypt                           | ✅        |
| Connector version semver check                                              | ✅        |
| Connector version increment check                                           | ✅        |
| QA checks                                                                   | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/30781-truncate-log/1697051995/50d8f7664daaefdf22109ade5de1877cf00762c3/source-mysql-strict-encrypt/3.1.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:50d8f7664daaefdf22109ade5de1877cf00762c3)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mysql-strict-encrypt test
```

**Pull request merged**